### PR TITLE
chore: correct typos (#589)

### PR DESCRIPTION
**Pull request by dongxuwang**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

- Correct all typos in current code

## Checklist

- []  I have written the necessary rustdoc comments.
- []  I have added the necessary unit tests and integration tests.

## Refer to a related PR or issue link (optional)
#589

### CODE REVIEW

File: src/common/query/src/signature.rs [hunk 1]
```diff
@@ -118,14 +118,14 @@ impl From<TypeSignature> for DfTypeSignature {
     fn from(type_signature: TypeSignature) -> DfTypeSignature {
         match type_signature {
             TypeSignature::Variadic(types) => {
-                DfTypeSignature::Variadic(concret_types_to_arrow_types(types))
+                DfTypeSignature::Variadic(concert_types_to_arrow_types(types))
```
Comment (waynexia) [inline src/common/query/src/signature.rs:121]: It should be `concrete` instead of `concert`

**Review by waynexia — COMMENTED** [review]
Thanks @dongxuwang !

File: src/common/query/src/signature.rs (hunk 1 above)
Comment (dongxuwang) [reply src/common/query/src/signature.rs]: Thank you, updated

**Review by waynexia — APPROVED** [review]
Thanks :heart:

**Review by v0y4g3r — APPROVED** [review]
LGTM

**Pull request merged**